schema/schema.go

```diff
@@ -0,0 +1,102 @@
+package schema
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/cloudfoundry/cli/cf/terminal"
+)
+
+type SchemaParser struct {
+	ui terminal.UI
+}
+
+func NewSchemaParser(userInterface terminal.UI) *SchemaParser {
+	return &SchemaParser{
+		ui: userInterface,
+	}
+}
+
+func (p *SchemaParser) ParseSchema(schema string) (string, error) {
```

Comment:
Dumb question time!

Is this implementing an existing external interface? If not, would it make sense for the `string` argument / return value to be `[]byte`s instead?